requirements.in

```diff
@@ -9,5 +9,4 @@ pydantic
 psycopg2-binary
 sqlalchemy
 raven[flask]
-f8a_worker @ git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@066c2f6#egg=f8a_worker
-f8a_utils @ git+https://github.com/fabric8-analytics/fabric8-analytics-utils.git@5a5ce60#egg=f8a_utils
+f8a_worker @ git+https://github.com/fabric8-analytics/fabric8-analytics-worker.git@b4e1fea#egg=f8a_worker
```

Comment:
Can't we completely eliminate worker dependency now?

Reply:
@arajkumar as of now worker is required for `WorkerResult` model, I am not much familiar with backbone code so not sure if we can use model from some other place.
https://github.com/fabric8-analytics/f8a-server-backbone/blob/1367cf92f4a31e4cccb8c6e4074c1e3d489cf53b/src/utils.py#L14

Reply:
Ok, then you can remove init calls to selinon right?

Reply:
Yes @arajkumar I checked this as well, it is used in V1. I have not touched anything in V1, so init_celery and init_selinon calls are still there.
https://github.com/fabric8-analytics/f8a-server-backbone/blob/1367cf92f4a31e4cccb8c6e4074c1e3d489cf53b/src/stack_aggregator.py#L671